docs/wiki/installation/cli-flags.md

```diff
@@ -102,10 +102,20 @@ If this value is >0 then the watchdog level (`--watchdog_level`) for maximum mem
 
 `--watchdog_utilization_limit=0`
 
-If this value is >0 then the watchdog level (`--watchdog_level`) for maximum sustained CPU utilization is overridden. Use this if you would like to allow the `osqueryd` process to use more than 30% of a thread for more than 9 seconds of wall time. The length of sustained utilization is not independently configurable.
+If this value is >0 then the watchdog level (`--watchdog_level`) for maximum sustained CPU utilization is overridden. Use this if you would like to allow the `osqueryd` process to use more than 10% of a thread for more than `--watchdog_latency_limit` seconds of wall time. The length of sustained utilization is configurable with `--watchdog_latency_limit`.
```

Comment:
This might be fine, but it also seems a little weird?

If the value isn't conforming (like, if it's under 0) we should throw an error, and not silently ignore it.

And it feels weird to have a command line that defaults to 0, and then have some other default somewhere. Why not have the command line default to what we want? (there might be history behind it, but it feels byzantine)

Reply:
It's maybe less weird in context, because there is a different default value depending on `--watchdog_level`, so there isn't really a single "default" value.

For example, "normal" CPU utilization limit is 10%, restrictive is 5%, and off is 100%.

Now... maybe the concept of "watchdog level" is outdated and could be replaced by having defaults for each of these values instead? But as is, it's pretty reasonable to have this set to 0 to distinguish between "use the value set by this flag" and "use the value for the configured watchdog level".

Reply:
Agree that this is weird, but it's consistent with the other flags. The weirdness is not net new. :P